pyproject.toml

```diff
@@ -5,11 +5,73 @@ requires = [
     "setuptools>=42",
 ]
 
-[tool.isort]
-force_sort_within_sections = true
-multi_line_output = 5
-profile = "hug"
-skip_gitignore = true
+[project]
+authors = [
+    {name = "Ian Thomas"},
+    {email = "ianthomas23@gmail.com"},
```

Comment:
Shouldn't `name` and `email` go in the same dictionary?

Reply:
I think I came across all possible permutations of these settings when I wrote this. But I agree that it would be better to put them both in the same dictionary. Thanks for pointing this out.